Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -626,7 +605,8 @@ void EmuCodeBlock::WriteToConstRamAddress(int accessSize, OpArg arg, u32 address
 	if (arg.IsImm())
 	{
 		arg = SwapImmediate(accessSize, arg);
-		MOV(accessSize, MDisp(RMEM, address & 0x3FFFFFFF), arg);
+		MOV(32, R(RSCRATCH), Imm32(address));
+		MOV(accessSize, MRegSum(RMEM, RSCRATCH), arg);
```

Comment:
ick, an extra instruction here. is there no way to pre-translate the address so that it'll fit in an int32_t offset?